src/comparison_diff.rs

```diff
@@ -0,0 +1,15 @@
+use colored::*;
```

Comment:
i dont think you need this file since this code can't really be shared :) i would just put it inside the assertion function directly

Reply:
i mean `greater_than.rs` file

Reply:
Got it. Fixing.

Reply:
Resolved.